styles.css

```diff
@@ -26,4 +26,12 @@ li a {
   padding: 20px 16px;
   text-decoration: none;
   font-family: "Courier New", Courier, monospace
+}
+
+section {
+  max-height: 800px;
+  max-width: 800px;
```

Comment:
It's functional without it, but it is your choice as the designer/developer. If you want to control it the width (e.g,. if your user's screen happens to be particularly wide), then this is the right property to use.

Optional update: center the content if you're going to limit the width. Implement at your discretion.